.dlc.json

```diff
@@ -0,0 +1,44 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^http://localhost"
+    },
+    {
+      "pattern": "^#"
+    },
+    {
+      "pattern": "^{"
+    },
+    {
+      "pattern": "^https://repo1.maven.org/maven2/org/apache/flink.*SNAPSHOT.*"
+    },
+    {
+      "pattern": "^https://mvnrepository.com"
+    },
+    {
+      "pattern": "^https://img.shields.io"
+    },
+    {
+      "pattern": "^https://tokei.rs"
+    },
+    {
+      "pattern": "^https://json.org/"
+    },
+    {
+      "pattern": "^https://opencollective.com"
+    },
+    {
+      "pattern": "^https://twitter.com*"
+    }
```

Comment:
Do we have these links in the documentation?

Reply:
I will polish this config.